### PR TITLE
Chore/add comms around removal of tce

### PR DESCRIPTION
**Pull request by joshenlim**
Replace the previously removed "Security" section of the Column side panel editor in the table editor when creating new columns to include some information around the removal of TCE GUI support

### CODE REVIEW

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx [hunk 1]
```diff
@@ -338,6 +350,46 @@ const ColumnEditor = ({
...
+            header={<FormSectionLabel className="lg:!col-span-4">Security</FormSectionLabel>}
+          >
+            <FormSectionContent loading={false} className="lg:!col-span-8">
+              <Alert_Shadcn_>
+                <IconAlertCircle />
+                <AlertTitle_Shadcn_>
+                  Column encryption has been removed from the GUI
+                </AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_>
+                  <p className="!leading-normal">
```
Comment (MildTomato) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx:367]: is the leading not normal by default ?

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx [hunk 2]
```diff
@@ -338,6 +350,46 @@ const ColumnEditor = ({
...
+                    You may still encrypt new columns through the SQL editor using{' '}
+                    <Link
+                      href={`/project/${ref}/database/extensions?filter=pgsodium`}
+                      className="text-brand hover:underline"
+                    >
+                      pgsodium's
+                    </Link>{' '}
+                    Transparent Column Encryption (TCE).
+                  </p>
+                  <Button type="default" icon={<IconExternalLink />} className="mt-2">
```
Comment (MildTomato) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx:377]: asChild ?

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx:367]: its set to leading-relaxed for p tags, unless i should be using another tag here?

image

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx (hunk 2 above)
Comment (joshenlim) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx:377]:
```suggestion
                  <Button asChild type="default" icon={<IconExternalLink />} className="mt-2">
```

**Review by alaister — APPROVED** [review]

**Pull request merged**